sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/src/Generated/Models/MsixImageURI.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+
+namespace Azure.ResourceManager.DesktopVirtualization.Models
+{
+    /// <summary> Represents URI referring to MSIX Image. </summary>
+    public partial class MsixImageURI
```

Comment:
Should be Uri, we can use the renaming rule to update this:
```yaml
rename-rules:
  URI: Uri
```

Reply:
the rename-rules can fix the model name but not work for the parameter name, @ArcturusZhang  will check it later

Reply:
let's checkin first, then first this naming later once the codegen fixed